databases/databases/databases.md

```diff
@@ -6,41 +6,43 @@ Databases are kind of hidden in the back of the web application so people treat
 
 Compared to a normal programming language like you've already learned, SQL (Structured Query Language), which is used to query databases, is a very simple syntax... there are only a small handful of verbs to learn.  What trips people up is that you need to be able to visualize in your head what it's going to be doing.  We'll spend a fair bit of time on SQL and databases because they're so fundamental, but for now we'll just cover enough to get you familiar with what's going on in there.
 
-### Learning Outcomes
-Look through these now and then use them to test yourself after doing the assignment
+### Learning Overview
+This section contains a general overview of topics that you will learn in this lesson.
 
-* What is a database?
-* What are relational databases?
-* How are relational databases different from XML?
-* What is SQL?
-* What is SQL used for?
-* How do you get all the records from a table in SQL?
-* How do you insert a record in SQL?
+- What a database is.
```

Comment:
Same comments re: list spacing from #24773

Reply:
Should be fixed now!